frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -324,10 +323,10 @@ class DashboardHeaderContainer extends Component<DashboardHeaderProps> {
       >
         {t`Cancel`}
       </Button>,
-      <DashboardButtonTooltip
-        disabled={!isSaveDisabled}
-        label={disabledSaveTooltip}
+      <Tooltip
         key="save"
+        label={disabledSaveTooltip}
+        disabled={!isSaveDisabled}
       >
         <ActionButton
```

Comment:
I think we need to add a span around this action button, because the tooltip is not rendering without it.

**Without**
![image](https://github.com/metabase/metabase/assets/22608765/d4e56731-fabb-431e-b47c-5ed75ae3665d)

**With**
![image](https://github.com/metabase/metabase/assets/22608765/a37a6177-c81c-447d-b7b2-6ab6dd2a24db)